src/reducers/sources.js

```diff
@@ -542,7 +542,7 @@ export function getRelativeSourcesForThread(
   state: OuterState,
   thread: string
 ): SourcesMap {
-  return getRelativeSources(state)[thread];
+  return getRelativeSources(state)[thread] || {};
```

Comment:
i feel like this should not be necessary

the strange thing is that i needed this because in SourcesTree, `state.sources.relativeSources` was empty, but... `dbg.store.getState().sources.relativeSources` was not. I am really confused by this

<img width="412" alt="screen shot 2019-01-02 at 9 06 45 am" src="https://user-images.githubusercontent.com/254562/50597935-223f9480-0e77-11e9-9d14-fe90036c6fd4.png">